docs/deployments/ASF-deployment-ci-cf.yml

```diff
@@ -0,0 +1,99 @@
+AWSTemplateFormatVersion: 2010-09-09
+
+Parameters:
+  CFBucket:
+    Description: AWS S3 bucket fo uploading CloudFormation templates
+    Type: String
+
+Resources:
+  CloudformationDeploymentRole:
+    Type: AWS::IAM::Role
+    Properties:
+      Policies:
+        - PolicyName: cloud-formation-deployment-policy
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Effect: Allow
+                Action:
+                  - ec2:*
+                  - s3:*
+                  - ecs:*
+                  - batch:*
+                  - events:*
+                  - logs:*
+                  - iam:*
+                  - lambda:*
+                  - ssm:GetParameters
+                  - apigateway:*
+                  - states:*
+                  - dynamodb:*
+                  - cloudwatch:*
+                  - sns:*
+                  - secretsmanager:*
+                  - kms:*
+                Resource: "*"
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          Action: sts:AssumeRole
+          Principal:
+            AWS: !Ref AWS::AccountId
+          Effect: Allow
```

Comment:
I prefer seeing `AssumeRolePolicyDocument` first, before `Policies`, since "who is this role for" is usually slightly more important to me than "what can this role do"

Reply:
Good idea -- looks like it's listed that way in AWS docs too